scripts/apm-server-load-test.js

```diff
@@ -167,9 +177,13 @@ function generateTransaction(spanCount) {
       sampled: spanCount != 0,
       breakdown,
       context: {
+        tags: {
```

Comment:
any reason for not adding to the transaction.session field?

Reply:
Mainly because it's not recognised by APM server at the moment, I've added it for future support.